.github/workflows/release.yml

```diff
@@ -126,7 +126,7 @@ jobs:
           fetch-depth: 0 # FIXME: Starport requires full repository
 
       - name: Build
-        run: docker build -f docker/cheqd_node/Dockerfile --no-cache -t cheqd-node  --build-arg UID=$(id -u) --build-arg GID=$(id -g) . # TODO: Get rid of UID and GID
+        run: docker build -f docker/mainnet/Dockerfile --no-cache -t cheqd-node  --build-arg UID=$(id -u) --build-arg GID=$(id -g) . # TODO: Get rid of UID and GID
```

Comment:
Wait, for me, it looks like a misunderstanding of the folder structure. Is this image intended to be used with the mainnet only?

Reply:
Already changed to use the same Dockerfile which is now under docker/Dockerfile.
The only different Dockerfile is the one meant for building the localnet image, currently residing in single_image_testnet/Dockerfile.

Reply:
Got it, my misunderstanding